core/src/i18n/en.yaml

```
@@ -514,6 +516,7 @@ cron-jobs:
   schedule: Schedule
   title: Cron Jobs
 custom-resource-definitions:
+  description: <0>Custom Resource Definition</0> is a file used to define a custom resource that allows you to extend the Kubernetes API to cover use cases that are not directly covered by core Kubernetes.
```

```suggestion
  description: <0>Custom Resource Definition</0> is an object used to define a custom resource that allows you to extend the Kubernetes API for use cases that are not directly covered by core Kubernetes.
```
Using the `for` you can get rid of the double `cover` and `to`.
Also, CRD is not necessarily a file, it can be defined in various ways, e.g. as a Go struct, via an HTTP call, etc.